packages/oc-gs-storage-adapter/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "oc-gs-storage-adapter",
+  "version": "1.0.0",
```

Comment:
Maybe we can set the version to 0.0.1 to start with? as discussed I'll try to publish this as a [prerelease](https://www.npmjs.com/package/lerna#--npm-tag-tagname) (`beta`)